README.md

```diff
@@ -24,3 +45,23 @@ General DALES copyright applies for any files part of the original DALES distrib
 Additional files provided in uDALES are copyrighted "the uDALES Team" and are marked as such at the beginning of each file.
 
 All files are licensed under the GNU General Public License. See [LICENSE.txt](LICENSE.txt).
+
+## References
+
+Heus, T., van Heerwaarden, C. C., Jonker, H. J. J., Siebesma, A. P., Axelsen, S., van den Dries, K., Geoffroy, O., Moene, A. F., Pino, D., Roode, S. R. de, & de Arellano, J. V.-G. (2010). Formulation of the dutch atmospheric large-eddy simulation (DALES) and overview of its applications. Geoscientific Model Development, 3(2), 415–444. https://doi.org/10.5194/gmd-3-415-2010
+
+Grylls, T., Cornec, C. M. A. L., Salizzoni, P., Soulhac, L., Stettler, M. E. J., & van Reeuwijk, M. (2019). Evaluation of an operational air quality model using large-eddy simulation. Atmospheric Environment: X, 3, 100041. https://doi.org/10.1016/j.aeaoa.2019.100041
+
+Grylls, T. (2020). Simulating air pollution in the urban microclimate.
+
+Grylls, T., & van Reeuwijk, M. (2021). Tree model with drag, transpiration, shading and deposition: Identification of cooling regimes and large-eddy simulation. Agricultural and Forest Meteorology, 298-299, 108288. https://doi.org/https://doi.org/10.1016/j.agrformet.2020.108288
+
+Oke, T. R., Mills, G., Christen, A., & Voogt, J. A. (2017). Urban climates. Cambridge University Press. https://doi.org/10.1017/9781139016476
+
+Suter, I. (2018). Simulating the impact of blue-green infrastructure on the microclimate of urban areas. https://doi.org/10.25560/78715
+
+Suter, I., Grylls, T., Sützl, B. S., & van Reeuwijk, M. (2021). uDALES 1.0.0: A large-eddy simulation model for urban environments. In Geoscientific Model Development. Copernicus GmbH.
+
+Sützl, B. S., Rooney, G. G., & van Reeuwijk, M. (2020). Drag distribution in idealized heterogeneous urban environments. Boundary-Layer Meteorology. https://doi.org/10.1007/s10546-020-00567-0
```

Comment:
BLM has now printed the paper, so the reference needs to change to this:
Sützl, B. S., Rooney, G. G., & van Reeuwijk, M. (2021). Drag distribution in idealized heterogeneous urban environments. Boundary-Layer Meteorology, 178, 225-248. https://doi.org/10.1007/s10546-020-00567-0
Sorry I know this needs to change everywhere now.

Reply:
I have I have updated and I will also update in the JOSS paper.

Reply:
Thanks!